webapp/src/components/viewHeader/viewHeaderGroupByMenu.tsx

```diff
@@ -33,7 +33,7 @@ const ViewHeaderGroupByMenu = React.memo((props: Props) => {
                                 style={{color: 'rgb(var(--center-channel-color-rgb))'}}
                                 id='groupByLabel'
                             >
-                                {groupByPropertyName}
+                                {groupByProperty?.name}
```

Comment:
This seemed to be technically the same as before but I wonder if `undefined` should be handled properly like rendering an empty string whenever it occurred instead of potentially showing `undefined` even for a short period of time.

Reply:
This is JSX so using undefined here will produce [empty result](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored).